SPECS-EXTENDED/rubygem-rspec/rubygem-rspec.spec

```diff
@@ -38,18 +40,24 @@ gem build %{gem_name}
 gem install -V --local --force --install-dir %{buildroot}/%{gemdir} %{gem_name}-%{version}.gem
 mkdir -p %{buildroot}%{gem_dir}
 cp -a .%{gem_dir}/* %{buildroot}%{gem_dir}/
+#add lib folder to buildroot from Source0
+cp -r lib/ %{buildroot}%{gem_instdir}/
```

Comment:
Are the contents of `lib` pre-built (came with the source) or have they been built as part of the package build? If they are pre-built, then we shouldn't be taking them.

Reply:
lib/ has two .rb files, rspec.rb and version.rb. They're part of the source, don't get built during package build

Reply:
Got it, I see what you mean, looks good.